modules/search/pages/default-analyzers-reference.adoc

```diff
@@ -0,0 +1,93 @@
+= Default Analyzers 
+:page-topic-type: reference
+:description: Use an analyzer to filter and modify search strings to improve matches for search results. 
+ 
+
+[abstract]
+{description}
+
+Analyzers contain:
+
+* xref:customize-index.adoc#character-filters[Character filters], which remove unwanted characters from search input.
+* xref:customize-index.adoc#tokenizers[Tokenizers], which separate input strings into individual tokens.
+* xref:customize-index.adoc#token-filters[Token filters], which modify tokens.
+
+When you xref:create-type-mapping.adoc[create a type mapping], you can choose a default analyzer for your type mappings, or xref:create-custom-analyzer.adoc[create your own].
+
+The following default analyzer options are available: 
+
+[cols="1,2"]
+|====
+|Analyzer |Description 
+
+|inherit | If you set an analyzer to `inherit`, the Search index component inherits the xref:set-advanced-settings.adoc#default-analyzer[default analyzer set for an index]. 
+
+|ar |The `ar` analyzer uses character filters, tokenizers, and token filters designed for Arabic language searches.
```

Comment:
Would this be clearer as just "Arabic"?
e.g. remove all the long duplicated text to focus on the important changing part.

Reply:
But it doesn't appear as "Arabic" in the UI - and I want to make it clear what the analyzer is doing. If someone isn't reading the whole page, I want them to be able to quickly CTRL + F and still get the information they need.

Reply:
OK 👍